### DISCUSSION TITLE
Proxied url's static resources not found

### DISCUSSION THREAD

**Discussion by chedges** [chimurai/http-proxy-middleware#579] [Q&A]
I'm trying to setup a proxy to from my main site to another url which has additional features (which will eventually replace the main site in time). I'm trying to make the transition seamless so the url in the browser shouldn't change so a redirect can't be used.

e.g. mymainsite.com/othersite is being proxied to theothersite.com via

`app.use('/othersite', createProxyMiddleware('http://theothersite.com', {logLevel: 'debug', changeOrigin: true}))`

I can see when I hit mymainsite.com/othersite in my browser, the index.html from theothersite.com is being served but the static files (js, css, images etc) required to make it render are not being served (404 error). the browser is looking for them on the mymainsite.com and not on theothersite.com where they would normally be found e.g.

_http://mymainsite.com/static/js/bundle.js_ but the actual file is found here _http://theothersite.com/static/js/bundle.js_

How should static resources like this be handled? is this even possible?

thanks in advance for any help and advice,

Chris.

**Comment by chimurai** [comment]
Your html might be using [absolute urls](https://stackoverflow.com/questions/2005079/absolute-vs-relative-urls).

Try modifying the html response and changing the absolute urls to absolute paths with [responseInterceptor](https://github.com/chimurai/http-proxy-middleware/blob/master/recipes/response-interceptor.md#response-interceptor).
